src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -934,6 +941,12 @@ protected override void HandleLaunchRequestAsync(IRequestResponder<LaunchArgumen
 
         protected override void HandleAttachRequestAsync(IRequestResponder<AttachArguments> responder)
         {
+            if (clientId == "visualstudio" && !responder.Arguments.ConfigurationProperties.GetValueAsBool("externalConsole").GetValueOrDefault(false))
+            {
+                responder.SetError(new ProtocolException("Integrated terminal is not supported in Visual Studio. To fix, set external console to true."));
+                return;
+            }
```

Comment:
I don't think attach requests have this property.

Reply:
Yep. My mistake of searching `externalConsole` and got 4 hits in the cpptools launch.json because of cppvsdbg. 

It is only in launch for cppdbg
https://github.com/microsoft/vscode-cpptools/blob/94294dd3728e034fbe2be6db0437baabcde4ef8b/Extension/package.json#L1452-L1456